### ISSUE TITLE
How can pretrainmed models be used with a different number of classes?

### ISSUE THREAD

**Issue by WaterKnight1998** [HRNet/HRNet-Semantic-Segmentation#136]
I haven't seen anyone working on that. 

I am interested in using pretrained models in a problem with different number of classes.

**Comment by jhultman** [comment]
You can use `torch.load` with `strict=False` to ignore shape mismatch. You may want to use smaller learning rate for early layers or freeze them.

**Comment by WaterKnight1998** [comment]
> You can use `torch.load` with `strict=False` to ignore shape mismatch. You may want to use smaller learning rate for early layers or freeze them.

I am using the method init_weights with the path to pretrained weights on imagenet. However, I decided to remove last_layer as it is the head and just use it in what could be called the backbone!
